### DISCUSSION TITLE
Navigation column on the ACG Website: long, mixed, or clean?

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#65] [Polls]
The backbone of the [ACG Website on GitHub](https://a-cosmology-group.github.io/acg/) is mostly established, we must now decide what it's going to look like.

Of concern is that the navigation column gets excessively long when it contains every page, like the large number of [Essays](https://a-cosmology-group.github.io/acg/essays/) in this example (and there are at least 6 more to be added...)

![navigation-long](https://github.com/a-cosmology-group/acg/assets/16748402/a61e5cf3-13ca-4948-9df3-bc66ffc3036d)

It is possible to have a mix of individual articles and *Table-of-Content* articles to decrease the clutter, as in [Cosmological Models](https://a-cosmology-group.github.io/acg/models/) which has approximately 75 articles!

![navigation-mix](https://github.com/a-cosmology-group/acg/assets/16748402/d77b23e3-32d1-4f9f-85a3-35d3c487e856)

Or, when a large number of items is present, only one page with a *table of contents* presents every page in the category, such as in [Redshift Models](https://a-cosmology-group.github.io/acg/redshift/) with 95 articles.  This last possibility requires a bit more work to set up but makes the navigation column cleaner.

![navigation-clean](https://github.com/a-cosmology-group/acg/assets/16748402/7204b91c-8baf-4ad0-82aa-fa46eb2be126)

**Comment by ftessier** [comment]
I would definitely side with links in the sidebar that leads to a toc page for that section. Note that in Markdown there are tools to generate the toc automagically!

**Reply by RedshiftDrift** [reply]
`just-the-docs` does that too, it's already implemented e.g. for [Essays](https://a-cosmology-group.github.io/acg/essays/)

**Reply by ftessier** [reply]
It looks really good!
